.ci/containers/changelog-checker/check_changelog.sh

```diff
@@ -0,0 +1,49 @@
+#! /bin/bash
+
+set -e
+
+pr_number=$1
+mm_commit_sha=$2
+build_id=$3
+project_id=$4
```

Comment:
https://cloud.google.com/build/docs/configuring-builds/substitute-variable-values

There are default substitutions that should be available.. I don't think you need to pass these to every op

Reply:
It seems like they're not automatically available to scripts; I can either pass them as args to the command, or I can pass them as env vars. I went with args initially to match what was happening with PR_NUMBER (and because I didn't realize env vars were an option.) I could do a separate PR to convert them all to env vars?

Or am I completely misunderstanding and they are automatically available inside scripts?

Reply:
They should be available inside scripts. These environment variables are available in each step.

Reply:
actually wait... I could be wrong here

Reply:
yes you are right